lib/_http_common.js

```diff
@@ -222,9 +222,55 @@ exports.httpSocketSetup = httpSocketSetup;
 /**
  * Verifies that the given val is a valid HTTP token
  * per the rules defined in RFC 7230
+ * See https://tools.ietf.org/html/rfc7230#section-3.2.6
+ *
+ * This implementation of checkIsHttpToken() loops over the string instead of
+ * using a regular expression since the former is up to 180% faster with v8 4.7
```

Comment:
This comment probably should also be updated to reflect the situation on v8 4.8 (that probably didn't change, but still).